src/Components/App.js

```diff
@@ -3,26 +3,42 @@ import '../CSS_styling/App.css';
 import movieData from '../movie-data';
 import Movies from './Movies';
 import NavBar from './NavBar';
+import Details from './Details'
 
 class App extends Component {
   constructor() {
    super()
    this.state = {
      movieInfo: movieData.movies
    }
-   console.log(this.state.movieInfo)
+   console.log("this.state.movieInfo here---", this.state.movieInfo)
+  }
+
+  findDetails = (event) => {
+    this.state.movieInfo.find(movie => {
+      let id = Number(event.target.id)
+      if (movie.id === id) {
+      this.setState({movieInfo: [movie]})
+      return movie;
+      }
+    })
   }
 
   render() {
     return (
       <>
         <NavBar />
-        <main>
-          <Movies movieInfos={this.state.movieInfo}/>
+        <main className='main-container'>
+        {this.state.movieInfo.length < 2 ? <Details movieInfo={this.state.movieInfo} /> :
```

Comment:
Very nice!! Great job getting this tricky conditional rendering set up. One thing you might want to change though is to move this out into a function. It can be pretty much the same code, but will live outside of your render function. Then, you can call it in render. Its a good way to keep your code clean!